INVENTORY.md

```diff
@@ -0,0 +1,22 @@
+# Inventory
+
+this page talks about common inventory information information provided by DPU vendors in a same format
+
+- Questions:
+  - Where does it run?
+    - service/container running on ARM cores?
+    - DPU firmware?
```

Comment:
what is FW? NIC fw? the FW or bootloaders are needed in case OS is not provisioned yet...